packages/nc-gui/composables/useViewFilters.ts

```diff
@@ -201,13 +275,15 @@ export function useViewFilters(
     reloadData?.()
   }
 
+  const saveOrUpdateDebounced = useDebounceFn(saveOrUpdate, 500)
+
   const addFilter = () => {
-    filters.value.push({ ...placeholderFilter })
+    filters.value.push({ ...placeholderFilter() })
     $e('a:filter:add', { length: filters.value.length })
   }
 
   const addFilterGroup = async () => {
-    const child = { ...placeholderFilter }
+    const child = { ...placeholderFilter() }
```

Comment:
wince each time it returns a new object you can remove the spread syntax ,`const child = placeholderFilter()` is enough